common/src/main/java/com/wynntils/services/itemfilter/statproviders/QualityStatProvider.java

```diff
@@ -17,11 +17,6 @@ public List<Integer> getValue(WynnItem wynnItem) {
         return List.of(qualityTierItemProperty.getQualityTier());
     }
 
-    @Override
-    public Class<Integer> getType() {
-        return Integer.class;
-    }
-
     @Override
     public List<String> getAliases() {
         return List.of("tier");
```

Comment:
This no longer works due to the tier stat

Reply:
fixed